src/core/util/sys_vars.cpp

```diff
@@ -692,7 +692,7 @@ void mce_sys_var::update_multi_process_params()
             rx_num_wr = 1;
             strq_strides_compensation_level = 32;
             strq_stride_size_bytes = 512;
```

Comment:
replace this value with STRQ_MIN_STRIDE_SIZE_BYTES as well

Reply:
Done, and tested.